src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/lib/embeddable_api.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export * from '../../../../../embeddable_api/public/np_ready/public';
```

Comment:
why do we have this ?

Reply:
This re-exports `embeddable` plugin's stateless code to `dashboard_embeddable_container` plugin so that the latter can always import `embeddable` code from a fixed location<sup>[1]</sup>.

---

<sup>1. Discussed that the imports that don't use this re-export are only tests and we want to keep this. </sup>